core/src/test/java/kafka/log/remote/RemoteLogManagerTest.java

```diff
@@ -3014,6 +3014,28 @@ public void testCopyThrottling() throws Exception {
         assertEquals(99L, capturedValues.get(1).longValue());
     }
 
+    @Test
+    public void testTierLagResetsToZeroOnBecomingFollower() {
```

Comment:
@kamalcph do you confirm the test was broken before the fix?

Reply:
yes